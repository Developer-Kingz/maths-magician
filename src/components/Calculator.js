/* eslint-disable */
import React from "react";
import "../App.css";
import Calculate from "./Calculate.js";

const Calculator = () => (
    <div className="main">
        <Calculate />
    </div>
);

export default Calculator;
/* eslint-enable */
