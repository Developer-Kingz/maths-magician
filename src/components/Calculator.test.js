import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import Calculator from "./Calculator";

describe("Calculator", () => {
    it("renders correctly", () => {
        const { asFragment } = render(
            <Router>
                <Calculator />
            </Router>
        );
        expect(asFragment()).toMatchSnapshot();
    });

    // Add more test cases here as needed.
});
