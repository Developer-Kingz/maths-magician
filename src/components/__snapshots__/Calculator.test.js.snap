// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <div>
      <div
        class="navbar"
      >
        <div
          class="logo"
        >
          <h1>
            Math Magician
          </h1>
        </div>
        <div
          class="nav"
        >
          <ul>
            <a
              aria-current="page"
              class="active"
              href="/"
            >
              <li>
                Home
              </li>
            </a>
            <a
              class=""
              href="/calculator"
            >
              <li>
                Calculator
              </li>
            </a>
            <a
              class=""
              href="/quotes"
            >
              <li>
                Quotes
              </li>
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="calc-container"
    >
      <h3>
        lets do some maths!
      </h3>
      <div
        class="calculator"
      >
        <div
          class="screen"
        >
          0
        </div>
        <div
          class="input-container"
        >
          <div>
            <button
              class="inputs"
              type="button"
            >
              AC
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              +/-
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              %
            </button>
          </div>
          <div>
            <button
              class="inputs-right"
              type="button"
            >
              /
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              7
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              8
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              9
            </button>
          </div>
          <div>
            <button
              class="inputs-right"
              type="button"
            >
              x
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              4
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              5
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              6
            </button>
          </div>
          <div>
            <button
              class="inputs-right"
              type="button"
            >
              -
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              1
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              2
            </button>
          </div>
          <div>
            <button
              class="inputs"
              type="button"
            >
              3
            </button>
          </div>
          <div>
            <button
              class="inputs-right"
              type="button"
            >
              +
            </button>
          </div>
          <div>
            <button
              class="zero-input"
              type="button"
            >
              0
            </button>
          </div>
          <div>
            <button
              class="dot-inputs"
              type="button"
            >
              .
            </button>
          </div>
          <div>
            <button
              class="equal-inputs"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
